spec/features/unauthenticated_user_cannot_access_admin_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Unauthenticated user', type: :feature do
+  create_brand
+  create_category
+  create_items
+    feature 'Access' do
+      it 'cannot checkout without logging in' do
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.